doc/source/user_guide/homs_api.rst

```diff
@@ -0,0 +1,76 @@
+.. _homs:
+
+:orphan:
+
+{{ header }}
+
+***************************
+pandas Higher Order Methods
+***************************
+
+pandas is experimenting with improving the behavior of higher order methods (HOMs). These
+are methods that take a function as an argument, often a user-defined function (UDF).
+They include ``.apply``, ``.agg``, ``.transform``, and ``.filter``. The goal is to make
```

Comment:
certainly could be done in followups

- link to the doc-strings for these methods
- in the doc-strings show the behavior changes when `use_hom` is enabled.

Reply:
Will link to doc strings here; going to hold off modifying the public doc-strings until this is ready.

Reply:
Done. I only listed methods that are currently being modified when using `api.use_hom=True`. Will expand the list as modifications occur.